modules/32_power/IntC/equations.gms

```diff
@@ -68,39 +69,37 @@ q32_limitCapTeStor(t,regi,teStor)$( t.val ge 2020 ) ..
   =l=
   sum(te2rlf(teStor,rlf),
     vm_capFac(t,regi,teStor)
-  * pm_dataren(regi,"nur",rlf,teStor)
```

Comment:
Value here was always 1 for storage technologies. I think it is find to remove it.